### ISSUE TITLE
PHoeNIx v2 does not include pigz in docker container

### ISSUE THREAD

**Issue by cvaske-clear** [theiagen/public_health_bioinformatics#216]
:bug:

### :pencil: Describe the Issue

Attempting to run PHoeNIx on a set of samples, from FASTQ, using the `"CDC_PHOENIX"` setting of the `phoenix_workflow` input. The Kraken2 database was the 

### :repeat: How to Reproduce

Terra on GCP, with dockstore pipeline `[github.com/CDCgov/phoenix/phoenix:v2.0.1](https://dockstore.org/workflows/github.com/CDCgov/phoenix/phoenix:v2.0.1)`

Kraken2 database was downloaded from https://benlangmead.github.io/aws-indexes/k2, specifically:

https://genome-idx.s3.amazonaws.com/kraken/k2_standard_08gb_20230605.tar.gz

And reuploaded into the workspace files area. A workspace variable was configured to point to the kraken2 URI, and the workflow run used that workspace variable.

(Sub-issue that I'm not sure where to file, the SOP for PHoeNIx 2 from Theiagen specifies a URI for the Kraken2 DB that is not publicly available, but perhaps that URI is only available to customers...)

### :fishing_pole_and_fish: Expected Behavior
Successful completion of the pipeline, without an error.

### :floppy_disk: Version Information

Dockstore URI: https://dockstore.org/workflows/github.com/CDCgov/phoenix/phoenix:v2.0.1

Kraken2 DB source URI: https://genome-idx.s3.amazonaws.com/kraken/k2_standard_08gb_20230605.tar.gz

### :information_source: Additional Information

This section of the `phoenix.log` file shows the error. `pigz` occurs in several parts of the shell script, but the command was not found.

```
Error executing process > 'CDC_PHOENIX:PHOENIX_EXQC:ASSET_CHECK (1)'

Caused by:
  Process `CDC_PHOENIX:PHOENIX_EXQC:ASSET_CHECK (1)` terminated with an error exit status (127)

Command executed:

  if [[ REFSEQ_20230504_Bacteria_complete.msh.gz = *.gz ]]
  then
      pigz -vdf REFSEQ_20230504_Bacteria_complete.msh.gz
  else
      :
  fi
  
  if [[ mlst_db.tar.gz = *.tar.gz ]]
  then
      tar --use-compress-program="pigz -vdf" -xf mlst_db.tar.gz
  else
      :
  fi
  
  if [[ k2_standard_08gb_20230605.tar.gz = *.tar.gz ]]
  then
      folder_name=$(basename k2_standard_08gb_20230605.tar.gz .tar.gz)
      tar --use-compress-program="pigz -vdf" -xf k2_standard_08gb_20230605.tar.gz
      mkdir ${folder_name}_folder
      mv *.kmer_distrib ${folder_name}_folder
      mv *.k2d ${folder_name}_folder
      mv seqid2taxid.map ${folder_name}_folder
      mv inspect.txt ${folder_name}_folder
      mv ktaxonomy.tsv ${folder_name}_folder
  else
      folder_name=$(basename k2_standard_08gb_20230605.tar.gz .tar.gz)
      mv ${folder_name} ${folder_name}_folder
  fi
  
  cat <<-END_VERSIONS > versions.yml
  "CDC_PHOENIX:PHOENIX_EXQC:ASSET_CHECK":
      phoenix_base_container: base_v2.0.0
  END_VERSIONS

Command exit status:
  127

Command output:
  (empty)

Command error:
  .command.sh: line 4: pigz: command not found
```

**Comment by cvaske-clear** [comment]
Apologies, filed this against the wrong repo!

**Comment by jvhagey** [comment]
Hi @cvaske-clear can you open this issue on the [phoenix repo](https://github.com/CDCgov/phoenix/issues), thanks!

**Comment by kevinlibuit** [comment]
@cvaske-clear do you mind sharing whar version of Phoenix you are running? This may have also been addressed in [v2.0.2](https://github.com/CDCgov/phoenix/releases/tag/v2.0.2)

**Comment by cvaske-clear** [comment]
Yes, using the non-latest version was the issue. Both the version and filing this bug here were the result of a miscommunication during a tag-team effort of running the pipeline.

The pipeline is now successfully running on Terra with v2.0.2 on our data.
